src/Collection.php

```diff
@@ -400,16 +400,16 @@ public function createSearchIndex($definition, array $options = []): string
      * For example:
      *
      *     $indexes = [
-     *         // Create a search index with the default name, on
+     *         // Create a search index with the default name on a single field
      *         ['definition' => ['mappings' => ['dynamic' => false, 'fields' => ['title' => ['type' => 'string']]]]],
      *         // Create a named search index on all fields
      *         ['name' => 'search_all', 'definition' => ['mappings' => ['dynamic' => true]]],
      *     ];
      *
      * @see https://www.mongodb.com/docs/manual/reference/command/createSearchIndexes/
      * @see https://mongodb.com/docs/manual/reference/method/db.collection.createSearchIndex/
-     * @param list<array{name?: string, definition: array|object}> $indexes List of search index specifications
-     * @param array{comment?: string}                              $options Command options
```

Comment:
This was originally typed as a `string` in https://github.com/mongodb/mongo-php-library/commit/f6ce2119fad23641df100248c11f081f15ad6cb5, despite `createSearchIndex()` correctly using `mixed` for its option.